src/Web/Slack/Chat.hs

```diff
@@ -106,11 +89,60 @@ mkPostMsgReq channel text =
     }
 
 data PostMsgRsp = PostMsgRsp
-  { postMsgRspTs :: String
+  { postMsgRspTs :: Text
   , postMsgRspMessage :: PostMsg
   }
   deriving stock (Eq, Generic, Show)
 
 instance NFData PostMsgRsp
 
 $(deriveFromJSON (jsonOpts "postMsgRsp") ''PostMsgRsp)
+
+-- | <https://api.slack.com/methods/chat.update>
+data UpdateReq = UpdateReq
+  { updateReqChannel :: ConversationId
+  , updateReqTs :: Text
```

Comment:
hmm, that's strange. do all messages in a channel have a unique timestamp?

Reply:
yes, lmao. They are the primary key of messages, which is an Interesting design decision on the part of Slack, but I guess, there are many of those.